src/pymor/algorithms/lradi.py

```diff
@@ -174,15 +172,14 @@ def projection_shifts_init(A, E, B, shift_options):
     shifts
         A |NumPy array| containing a set of stable shift parameters.
     """
+    random_state = get_random_state(seed=shift_options['init_seed'])
     for i in range(shift_options['init_maxiter']):
         Q = gram_schmidt(B, atol=0, rtol=0)
         shifts = spla.eigvals(A.apply2(Q, Q), E.apply2(Q, Q))
         shifts = shifts[shifts.real < 0]
         if shifts.size == 0:
             # use random subspace instead of span{B} (with same dimensions)
-            if shift_options['init_seed'] is not None:
-                np.random.seed(shift_options['init_seed'] + i)
-            B = B.space.from_numpy(np.random.randn(len(B), B.space.dim))
+            B = B.random(len(B), distribution='normal', random_state=random_state)
```

Comment:
Is `space` missing?
```suggestion
            B = B.space.random(len(B), distribution='normal', random_state=random_state)
```

Reply:
No, similar to `zero` or `empty`, there is a shorthand for `random` in `VectorArrayInterface`.